sklearn/cluster/_optics.py

```diff
@@ -358,13 +358,13 @@ def compute_optics_graph(X, *, min_samples, max_eps, metric, p, metric_params,
         as a core point. Expressed as an absolute number or a fraction of the
         number of samples (rounded to be at least 2).
 
-    max_eps : float, optional (default=np.inf)
+    max_eps : float, default=np.inf
         The maximum distance between two samples for one to be considered as
         in the neighborhood of the other. Default value of ``np.inf`` will
         identify clusters across all scales; reducing ``max_eps`` will result
         in shorter run times.
 
-    metric : string or callable, optional (default='minkowski')
+    metric : string or callable, default='minkowski'
```

Comment:
Python basic types (`str` instead of `string`).

Reply:
ok